documentation/guides/draft/use-cases/end-to-end-encryption-with-kafka/README.md

```diff
@@ -1,6 +1,136 @@
-# End-to-end encrypted messaging with Ockam and Kafka
+# End-to-end encrypted messaging with Kafka and Ockam Secure Channels
+
+## Background
+
+When using Apache Kafka and other stream processing services a common concern is data access and security.
+
+A lot of fields like finance and personal information handling require or advise data to be encrypted when stored and transferred.
+
+Since modern networks and data pipelines involve multiple steps when data is transferred or stored, it becomes more apparent that just encoding data in transit between two endpoints or while in storage does not provide enough security.
```

Comment:
Modern networks and data pipelines transfer data through multiple endpoints, often controlled by multiple vendors. It is therefore not sufficient for encryption to only cover the transit between arbitrary pairs of endpoints or once data comes to rest in storage.